BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/VerifiedAccount.java

```diff
@@ -0,0 +1,45 @@
+package nl.tudelft.ewi.ds.bankchain;
+
+/**
+ * @author Isha Dijcks
+ */
+public class VerifiedAccount {
+
+    private String publicKey;
```

Comment:
You should then add a Date.today() or something like that to BunqTransactionParser.